### PR TITLE
Uplink item's main entry now states if it has a discount

### LINKED ISSUE #28286: Discounted uplink gear still costs the full amount if you buy it from elsewhere in the menu
I felt a bit cheated buying a stetchkin for 7tc, only to check the discounted gear section later and see it available for only 3tc

### PR DESCRIPTION
**Pull request by Cyberboss**
Fixes #28286

### CODE REVIEW

File: code/modules/uplink/uplink_item.dm [hunk 1]
```diff
@@ -47,6 +47,7 @@ GLOBAL_LIST_EMPTY(uplink_items) // Global list so we only initialize this once.
 		var/discount = A.get_discount()
 		var/list/disclaimer = list("Void where prohibited.", "Not recommended for children.", "Contains small parts.", "Check local laws for legality in region.", "Do not taunt.", "Not responsible for direct, indirect, incidental or consequential damages resulting from any defect, error or failure to perform.", "Keep away from fire or flames.", "Product is provided \"as is\" without any implied or expressed warranties.", "As seen on TV.", "For recreational use only.", "Use only as directed.", "16% sales tax will be charged for orders originating within Space Nebraska.")
 		A.limited_stock = 1
+		I.name += " (Available at a lower price!)"
```
Comment (ChangelingRain) [inline code/modules/uplink/uplink_item.dm]: add it to the desc! the name is already kind of a bad spot, since it's used for logging.

Comment (KorPhaeron) [inline code/modules/uplink/uplink_item.dm]: Most older players probably don't read the desc though, so maybe there is another place it can be put

**Comment by PKPenguin321** [comment]
Add it to the actual purchase button?

**Comment by DoctoreMad** [comment]
Pseudo feature freeze reeeeee

(I'm actually fine with this because it seems retarded)

File: code/modules/uplink/uplink_item.dm (hunk 1 above)
Comment (Iamgoofball) [inline code/modules/uplink/uplink_item.dm]: tag it with a star tbh

**Comment by WJohn** [comment]
Not really optimal... I'd prefer if it just listed the discount price in the regular categories, with a before (now cross out) price, and an after price in a special font (maybe colored, bolded, exclamation mark etc). It'd be a lot more slick looking.

**Comment by Cyberboss** [comment]
That would make the entire entry limited stock than

**Comment by Tacolizard** [comment]
I don't think the freeze affects this because it isn't adding any new items

**Comment by lzimann** [comment]
Still doing it @Cyberboss ?

**Comment by Cyberboss** [comment]
nope

**Pull request closed without merging**